source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +164,238 @@ Procedure
+            ``config.json`` file. The file should resemble the following:
+
+            .. code-block:: json
+
+               [
+                  {
+                     "name": "<Error Handler Function Name>",
+                     "private": <boolean>
+                  }
+               ]
+
+            For more information on function configuration files, 
+            see :ref:`Function Configuration Files <appconfig-functions>`.
+
+         .. step:: Add an Error Handler to Your Trigger Configuration
+
+            Add an ``error_handler`` attribute to your trigger configuration file
+            in the ``Triggers`` folder. The trigger configuration file should 
+            resemble the following:
+
+            .. code-block:: json
+
+                     {
+                     "name": "...",
+                     "type": "...",
+                     "event_processors": {
+                        "AWS_EVENTBRIDGE": {
+                              "config": {
+                                 "account_id": "<AWS Account ID>",
+                                 "region": "<AWS Region>",
+                                 "extended_json_enabled": <boolean>
+                              }
+                           }
+                        },
+                        "error_handler": {
+                           "config": {
+                              "enabled": <boolean>,
+                              "function_name": "<Error Handler Function Name>"
+                           }
+                        }
+                     }
+            
+            For more information on trigger configuration files, see
+            :ref:`Trigger Configuration Files <appconfig-triggers>`.
+
+.. _eventbridge-error-handler-parameters:
+
+Error Handler Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default error handler has two parameters: ``error`` and ``changeEvent``.
+
+``error`` 
+`````````
+Has two attributes: ``code`` and ``message``.
+
+- ``code``: For a list of error codes 
+used by the error handler, see the below section on :ref:`Error 
+Codes <eventbridge-error-codes>`. 
+
+- ``message``: The unfiltered error message from an errored Eventbridge
+put request.
+
+``changeEvent``
+```````````````
+
+The requested change to your data made by EventBridge. For more information
+on types of change events and their configurations, see 
+:manual:`Change Events </reference/change-events>`.
+
+.. _eventbridge-error-codes:
+
+Error Codes
+~~~~~~~~~~~
+
+If an error was recevied from EventBridge, the event processor will parse the 
+error as either ``DOCUMENT_TOO_LARGE`` or ``OTHER``. This parsed error is passed
+to the error handle function. 
+
+``DOCUMENT_TOO_LARGE``
+``````````````````````
+If the put entry for an EventBridge trigger event is larger
+than 256 KB, Eventbridge will throw an error. The error will either contain:
+
+1.  `status code: 400 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400>`_ and 
```

Comment:
this isn't the correct number list formatting - need to fix

Reply:
What is the correct format?